index.html

```diff
@@ -36,3 +36,136 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
+        </div>
+        <p class="card-header">Tonic</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(4).png" alt="firstoject">
+        </div>
+        <p class="card-header">Multi-Post<br>Stories</p>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+
+
+    <section class="about-me">
+        <h1 class="about-header">About<br> Myself</h1>
+        <p class="about-para"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+        <ul class="social-media">
+            <li class="fa fa-twitter fa-2x"></li>
+            <li class="fa fa-linkedin fa-2x"></li>
+            <li class="fa fa-medium fa-2x"></li>
+            <li class="fa fa-github fa-2x"></li>
+            <li class="fa fa-facebook fa-2x"></li>
+           </ul>
+        <button class=" btn1 btn2">Get my resume</button>
+        <h2 class="item-header">Languages</h2>
+        <ul class="langua">
+            <li class="lang-item">
+                <img src="img/Ellipse1.png" alt="lang-icons">
+                <p>JavaScript</p>
+            </li>
+            <li class="lang-item">
+                <img src="img/Ellipse1(1).png" alt="lang">
+                <p>HTML</p>
+            </li>
+            <li class="lang-item">
+                <img src="img/Ellipse1(2).png" alt="lang"> 
+                <p>CSS</p> 
+            </li>
+        </ul>
+        <h2 class="item-header">Frameworks</h2>
+        <ul>
+            <li>tool1</li>
+            <li>tool2</li>
+            <li>tool3</li>
+        </ul>
+        <h2 class="item-header">Skills</h2>
+        <ul>
+            <li>skill1</li>
+            <li>skill2</li>
+            <li>skill3</li>
+        </ul>
```

Comment:
Languages, Frameworks, and Skills have arrows in front. They should look the same as in the design. Please see the design again and make them exactly the same. Moreover please use the same font family as in the design.